aws/table_aws_ecr_image_findings.go

```diff
@@ -0,0 +1,134 @@
+package aws
+
+import (
+	"context"
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ecr"
+	"github.com/turbot/steampipe-plugin-sdk/v4/grpc/proto"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin"
+	"github.com/turbot/steampipe-plugin-sdk/v4/plugin/transform"
+	"time"
+)
+
+//// TABLE DEFINITION
+
+func tableAwsEcrImageFindngs(_ context.Context) *plugin.Table {
+	return &plugin.Table{
+		Name:        "aws_ecr_image_findings",
+		Description: "AWS ECR Image findings",
+		List: &plugin.ListConfig{
+			Hydrate: getAwsEcrImageScanningFindings,
+			IgnoreConfig: &plugin.IgnoreConfig{
+				ShouldIgnoreErrorFunc: isNotFoundError([]string{"RepositoryNotFoundException", "ImageNotFoundException", "ScanNotFoundException"}),
+			},
+			KeyColumns: []*plugin.KeyColumn{
+				{Name: "image_tag", Require: plugin.Required},
+				{Name: "repository_name", Require: plugin.Required},
+			},
+		},
+		GetMatrixItemFunc: BuildRegionList,
+		Columns: awsRegionalColumns([]*plugin.Column{
+			{
+				Name:        "repository_name",
+				Description: "The name of the repository.",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "image_tag",
+				Description: "The image tag",
+				Type:        proto.ColumnType_STRING,
+			},
+			{
+				Name:        "image_scan_completed_at",
+				Description: "The date and time, in JavaScript date format, when the repository was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "vulnerability_source_updated_at",
+				Description: "The date and time, in JavaScript date format, when the repository was created.",
+				Type:        proto.ColumnType_TIMESTAMP,
+			},
+			{
+				Name:        "finding",
+				Description: "Finding",
+				Type:        proto.ColumnType_JSON,
+				Transform:   transform.FromField("Finding"),
+			},
+		}),
+	}
+}
+
+// // LIST FUNCTION
+func getAwsEcrImageScanningFindings(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (interface{}, error) {
```

Comment:
```suggestion
func getAwsEcrImageScanningFindings(ctx context.Context, d *plugin.QueryData, _ *plugin.HydrateData) (any, error) {
```

Reply:
im note sure with this :p

Reply:
It's the same in the codebase. So.... don't fix if it works :)
Thanks @villers